packages/neuron-wallet/src/services/cells.ts

```diff
@@ -42,7 +44,7 @@ export default class CellsService {
 
   public static getDaoCells = async (
     lockHashes: string[],
-  ): Promise<Cell[]> => {
+  ): Promise<Output[]> => {
```

Comment:
Please change name accordingly now that it has a different return type.

Reply:
`import Cell from xxx` ?

Reply:
Renamed to `Cell` in this file.